api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Could you please add missing `final` keywords and also replace mocking by the following pattern. It tends to be more common syntax and we try to use it in all the tests right now.

```
# bad
when().doSomething().thenReturn();

# good
doReturn().when().doSomething();
```